Naming/src/main/java/com/epam/engx/cleancode/naming/task1/DeliveryOrderService.java

```diff
@@ -1,29 +1,36 @@
 package com.epam.engx.cleancode.naming.task1;
 
-import com.epam.engx.cleancode.naming.task1.thirdpartyjar.*;
+
+import com.epam.engx.cleancode.naming.task1.thirdpartyjar.DeliveryService;
+import com.epam.engx.cleancode.naming.task1.thirdpartyjar.Order;
+import com.epam.engx.cleancode.naming.task1.thirdpartyjar.OrderFulfilmentService;
+import com.epam.engx.cleancode.naming.task1.thirdpartyjar.Product;
+import com.epam.engx.cleancode.naming.task1.thirdpartyjar.NotDeliverableOrderException;
 
 import java.util.List;
 
-public class DeliveryOrderService implements IOrderService {
+public class DeliveryOrderService implements OrderService {
+    private DeliveryService orderTracker;
+    private OrderFulfilmentService contractor;
 
-    private DeliveryService mDeliveryService;
+    public void setOrderTracker(DeliveryService orderTracker) {
+        this.orderTracker = orderTracker;
+    }
 
-    private OrderFulfilmentService mOrderFulfilmentService;
+    public void setContractor(OrderFulfilmentService contractor) {
+        this.contractor = contractor;
+    }
 
-    public void submitOrder(Order pOrder) {
-        if (mDeliveryService.isDeliverable()) {
-            List<Product> products = pOrder.getProducts();
-            mOrderFulfilmentService.fulfilProducts(products);
+    @Override
+    public void submitOrder(Order order) {
+        if (orderTracker.isDeliverable()) {
+            contractor.fulfilProducts(getProduct(order));
         } else {
             throw new NotDeliverableOrderException();
         }
     }
 
-    public void setDeliveryService(DeliveryService pDeliveryService) {
-        this.mDeliveryService = pDeliveryService;
-    }
-
-    public void setOrderFulfilmentService(OrderFulfilmentService pOrderFulfilmentService) {
-        this.mOrderFulfilmentService = pOrderFulfilmentService;
+    private List<Product> getProduct(Order order) {
```

Comment:
```suggestion
    private List<Product> getProducts(Order order) {
```
Because it is a List

But I think that 
`getProducts(order)`
is very similar to
`order.getProducts()`
so it is unnecessary to extract new method